tests/test_full.yml

```diff
@@ -4,11 +4,11 @@
     - test_full
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_genotype_WT_vs_KO.tsv
-      md5sum: 93fd663cb976c102144e1ffb34d88e53
+      md5sum: b19e34f880a7cb14e031a00d6c8f1b74
```

Comment:
Please just remove the md5sums if they are not kept constant

Reply:
Here, the 5sum change was expected. I copied the test.yml and renamed it to test_full.yml, so the old 93f... sum is from test.yml, but the two configs have a different parameter that causes the results to change, so this is fine actually